test/postgresql.transaction.test.js

```diff
@@ -109,6 +109,33 @@ describe('transactions', function() {
     it('should see the committed insert', expectToFindPosts(post, 1));
   });
 
+  describe('on the model', function() {
+    var p1Content = {title: 'p1', content: 'post-a'};
+    var p2Content = {title: 'p2', content: 'post-a'};
+
+    before(function(done) {
+      Post.create(p1Content, function(err, p1) {
+        Post.create(p1Content, function(err, p2) {
```

Comment:
sorry it was changed in the old PR, but everything messed up due to the cherry-pick history and I create this one, please hold on review, I need to change something.